tests/cases/standalone/common/flow/flow_basic.result

```diff
@@ -1382,31 +1388,32 @@ ADMIN FLUSH_FLOW('requests_long_term');
 SELECT
     *
 FROM
-    requests_without_ip;
+    requests_without_ip
+ORDER BY ts ASC;;
```

Comment:
Maybe we only need to add `order by` to unstable results? Most results usually won't change.

Reply:
> Maybe we only need to add `order by` to unstable results? Most results usually won't change. This PR modifies too many sqlness tests.

The result of `select` is always not guaranteed to be sorted, I think it should be mandatory to add `ORDER BY` in sqlness cases.